pkg/kafsar/kafsar.go

```diff
@@ -33,16 +34,26 @@ type PulsarConfig struct {
 	TcpPort  int
 }
 
+type KafsarConfig struct {
+	MaxConsumersPerGroup     int
```

Comment:
I think these configs are belong to `KafkaConfig`. We need to take a look at kafka broker config files.